docs/docusaurus/create_docusaurus_website.sh

```diff
@@ -17,12 +17,14 @@ function exit_timeout() {
   docker-compose logs docusaurus
   echo ''
   echo "Timed out after ${1}s waiting for Docusaurus container to build. See logs above for more info."
+  echo "Possible remedies:"
+  echo "  - Remove node_modules directory (rm -Rf node_modules) and try again."
```

Comment:
Make this single quotes for defense-in-depth since there's a `rm -rf` inside (!!)